src/swoole/Swoole/Http/Response.php

```diff
@@ -11,13 +11,13 @@
 
 class Response
 {
-    public $fd;
+    public int $fd;
 
     public $socket;
```

Comment:
Maybe here we could add `Swoole\Coroutine\Socket` type hint?

Reply:
I don't really know the type of it

Reply:
Yeah, I just assumed it was that. Those were just suggestions. All good for me at this point.